snforge_std/src/trace.cairo

```diff
@@ -2,48 +2,55 @@ use core::result::ResultTrait;
 use core::starknet::testing::cheatcode;
 use core::starknet::ContractAddress;
 
+/// Tree-like structure which contains all of the calls and sub-calls along with the results
```

Comment:
```suggestion
/// Tree-like structure which contains all of starknet calls and sub-calls that happened in a test
```

Reply:
I don't like this definition because it's not emphasising the recursiveness of the struct, it appears then like it's top-level only. Also, why remove the info about the results?

Reply:
My only concern is that we should emphasise that it is a starknet call, not a regular function call